File: dotnet/SmartPlag.Tokenzier.CSharp/Model/StudentResult.cs
using System.Collections.Generic;

namespace SmartPlag.Tokenzier.CSharp.Model
{
  public class StudentResult
  {
    public StudentResult()
    {
      this.Files = new List<StudentTokenMap>();
    }

    public StudentResult(string firstName, string lastName)
      : this()
    {
      this.FirstName = firstName;
      this.LastName = lastName;
    }

    public string FirstName { get; private set; }
    public string LastName { get; private set; }
    public List<StudentTokenMap> Files { get; set; }

  }
}

File: java/Java8Tokenizer/src/com/smartplag/tokenizer/java/model/StudentResult.java
package com.smartplag.tokenizer.java.model;

import java.util.List;

import com.fasterxml.jackson.annotation.JsonProperty;

public class StudentResult {
	@JsonProperty("FirstName")
	private String firstName;
	@JsonProperty("LastName")
	private String lastName;
	@JsonProperty("Files")
	private List<TokenMap> files;

	public String getFirstName() {
		return firstName;
	}

	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}

	public String getLastName() {
		return lastName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

	public List<TokenMap> getFiles() {
		return files;
	}

	public void setFiles(List<TokenMap> files) {
		this.files = files;
	}

}
